jetty-core/jetty-http/src/main/java/org/eclipse/jetty/http/MimeTypes.java

```diff
@@ -37,6 +37,43 @@
 public class MimeTypes
 {
     static final  Logger LOG = LoggerFactory.getLogger(MimeTypes.class);
+    private static final Index<Locale> KNOWN_LOCALES;
+
+    static
+    {
+        Index.Builder<Locale> builder = new Index.Builder<>();
+        builder = builder.caseSensitive(false);
+        for (Locale locale : Locale.getAvailableLocales())
+            builder.with(locale);
+        KNOWN_LOCALES = builder.build();
+    }
+
+    /**
+     * @param languageDashCountry The language and optional country of the local. For example "en" or "un-uk".
```

Comment:
I guess you meant `en-uk` instead of `un-uk`. But that doesn't even exist, the right form is actually `en-gb`.